src/app.jsx

```diff
@@ -50,6 +50,7 @@ class Application extends React.Component {
             dropDownValue: 'Everything',
             notifications: [],
             showStartService: true,
+            version: '1.3.0',
```

Comment:
This looks very arbitrary. Could it default to `null,` or at least 0.0.0?

Reply:
it is the version we require from spec file, so I think it is a safe and reasonable default.